### PR TITLE
Java

### LINKED ISSUE #2873: [Java 15.2]: Catching Exception
### Description

Add audio, documentation, and video regarding the **Catching Exceptions**

---

**Location of File to be added**
The files should be added inside the *`Exception Types`* which will be inside *`Catching Exception`* folder which will be inside of Java folder. If the folder is not present, create the folder with this exact name.

- The documentation file can be in the `.md/.py/.ipynb` format.
- Audio should be in .mp3 format and to be submitted through a Google drive link.
- Video can be in any format and should be submitted through a Google Drive link.

Note:

You are required to comment with `\assign CONTENT_TYPE`, so that other content types can be created at the same time.
Failing to comment in the above manner will lead to the removal of assignment from the issue even if the bot assigns.
You are required to do a PR within 7 days otherwise you will be unassigned and others will be assigned.
Issues will be assigned on a first come first serve basis.

### Domain

Java_Domain

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by astharaj300**
<hr>

## Description 📜

Here I have explained about Catching Exceptions.
Fixes #2873 

<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Java_#Domain

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by astharaj300** [comment]
@AkMo3 please review my pr.

File: Java/Exception Handling/Catching_Exceptions/Catching_Exception.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# **Catching Exception in Java**
+## What is Exception?
+Exception in Java relate to fundamental *errors* that violate the rules of the Java Language, it can happen during the execution of a program and disrupts its normal flow.
+
+When an exceptional condition arises, an object representing that exception is created and *thrown* in the method that caused the error. That method may choose to handle the exception itself, or pass it on.Either way, at some point, the exception is *caught* and processed.
+
+* Java exception handling is managed via five keywords : *try*, *catch*, *throw*, *throws*, and *finally*.
+
+## What is **throwing the exception**?
+When the exception occurs in a method, the process of creating the exception object and handling it to the runtime environment is said to be **throwing the exception**.
```
Comment (AkMo3) [inline Java/Exception Handling/Catching_Exceptions/Catching_Exception.md:10]: `handling` -> `handing`

File: Java/Exception Handling/Catching_Exceptions/Catching_Exception.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            System.out.println(e.getMessage() + " occured, kindly check your code");
+        }
+        System.out.println("This is very important:");
+        
+    }
+}
+```
+**OUTPUT:**
+
+/ by zero occured, kindly check your code
```
Comment (AkMo3) [inline Java/Exception Handling/Catching_Exceptions/Catching_Exception.md:58]: `occured` -> `occurred`

File: Java/Exception Handling/Catching_Exceptions/Catching_Exception.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+The throw keyword is used to manually throw an exception from a method or any block, because the system generated exceptions are automatically thrown by the Java run-time system.
+
+General form of throw is :
+```java
+throw <Exception Ref>;
+```
+__Throws :__
+
+Any code that is thrown out of a method must be specified as such by a *throws* clause. It is used in the signature of method to indicate that this method might throw one of the listed type exceptions. The caller to thse methods has to handle the exception using a try-catch block.
```
Comment (AkMo3) [inline Java/Exception Handling/Catching_Exceptions/Catching_Exception.md:83]: `thse` -> `these`

**Review by AkMo3 — CHANGES_REQUESTED** [review]

**Comment by astharaj300** [comment]
@AkMo3 I have made the changes, please check it once

**Comment by garvitraj** [comment]
@AkMo3 things are resolved.
GTG

**Pull request merged**